src/public/media.ts

```diff
@@ -390,12 +424,30 @@ export namespace media {
     if (!callback) {
       throw new Error('[select Media] Callback cannot be null');
     }
+
     ensureInitialized(FrameContexts.content, FrameContexts.task);
     if (!isAPISupportedByPlatform(mediaAPISupportVersion)) {
       const oldPlatformError: SdkError = { errorCode: ErrorCode.OLD_PLATFORM };
       callback(oldPlatformError, null);
       return;
     }
+
+    if (isMediaCallForVideoAndImageInputs(mediaInputs)) {
+      if (
+        GlobalVars.hostClientType === HostClientType.desktop ||
+        GlobalVars.hostClientType === HostClientType.web ||
+        GlobalVars.hostClientType === HostClientType.rigel
+      ) {
```

Comment:
can we check if hostClientType is mobile since in future we can have more non mobile host type.

Reply:
Makes sense, updated the check.